src/ui/states/state-needs-payment-info.svelte

```diff
@@ -89,257 +124,191 @@
     }
   }
 
-  onMount(async () => {
-    const stripePk = paymentInfoCollectionMetadata.data.publishable_api_key;
-    const stripeAcctId = paymentInfoCollectionMetadata.data.stripe_account_id;
+  onMount(() => {
+    updateStripeVariables();
 
-    if (!stripePk || !stripeAcctId) {
-      throw new Error("Stripe publishable key or account ID not found");
-    }
+    window.addEventListener("resize", onResize);
 
-    stripe = await loadStripe(stripePk, {
-      stripeAccount: stripeAcctId,
-    });
+    return () => {
+      window.removeEventListener("resize", onResize);
+    };
+  });
+
+  onMount(() => {
```

Comment:
We have another `onMount` above, does it make sense to consolidate them?

Reply:
I think it's preferable to keep them separated